examples/machine-learning/nemo-framework/README.md

```diff
@@ -0,0 +1,55 @@
+README
+======
+
+1. Set up NeMo Framework Container
+
+This makes a few environment variable modifications to the
+nemo:23.11.framework container, and submits a slurm job to copy the framework
+launcher scripts and a few other auxilery files into your working directory. 
+
+	sbatch setup_nemo.sh
```

Comment:
The behavior of large indents differs for single line and multiline strings. I suggest just using triple-backtick code blocks.

Reply:
Fixed